reference/forms/types/options/empty_data.rst.inc

```diff
@@ -1,7 +1,7 @@
 empty_data
 ~~~~~~~~~~
 
-**type**: ``mixed``
+**type**: ``string`` or ``array`` when the form is compound
```

Comment:
Sadly this change is wrong. The data here is indeed mixed (at least for now, with current supported versions), since it just needs to match the view data format which can be anything.

Reply:
The default is a callable which returns `new DataClassOption()`

Reply:
Where I can found information about DataClassOption?

Reply:
The default is a callable which returns `mixed`. It depends of many variables:
https://github.com/symfony/symfony/blob/0924c001f988d736f1b6241453b7e3ef56c288d1/src/Symfony/Component/Form/Extension/Core/Type/FormType.php#L132-L144